package.json

```diff
@@ -46,6 +46,21 @@
 		"validate": "npm ls",
 		"version": "git add package.json && git commit -m \"Bump package version\""
 	},
+	"husky": {
+		"hooks": {
+			"pre-commit": "lint-staged"
+		}
+	},
+	"lint-staged": {
+		"*.{js,jsx}": [
+			"npm run lint:fix",
+			"git add"
+		],
+		"*.{js,jsx,ts,tsx,css,scss}": [
+			"prettier --write",
```

Comment:
The task above (`npm run lint:fix`) already covers `prettier --write` for js and jsx files.  Was this added for css?  I don't think this redundancy is needed.  Would this simpler config work?
```
"lint-staged": {
		"*.{js,jsx}": [
			"npm run lint:fix"
		]
	}
```